TINY-8568: Fix incorrect copying of colgroup table with colspans/rowspans (#7696)
tinymce/tinymce: modules/snooker/CHANGELOG.md

## Unreleased

### Fixed

> - `CopySelected` was incorrectly extracting a selected cells table for tables with colgroups.

Reviewer: This doesn't read well, or I can't seem to make sense of it. Can we please have a second go at the wording here?

Reviewer: Maybe something like this?
```suggestion
- The `CopySelected` module was incorrectly extracting the selected cells in a table that contained colgroups.
```
